charts/cluster-api-runtime-extensions-nutanix/README.md

```diff
@@ -32,12 +32,16 @@ A Helm chart for cluster-api-runtime-extensions-nutanix
 | deployment.replicas | int | `1` |  |
 | env | object | `{}` |  |
 | helmAddonsConfigMap | string | `"default-helm-addons-config"` |  |
-| helmRepositoryImage.pullPolicy | string | `"IfNotPresent"` |  |
-| helmRepositoryImage.repository | string | `"ghcr.io/nutanix-cloud-native/caren-helm-reg"` |  |
-| helmRepositoryImage.tag | string | `""` |  |
-| helmRepositorySecurityContext.fsGroup | int | `65534` |  |
-| helmRepositorySecurityContext.runAsGroup | int | `65534` |  |
-| helmRepositorySecurityContext.runAsUser | int | `65534` |  |
```

Comment:
I had concerns changing these values being a breaking change in a different PR from https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pull/947#discussion_r1811307404 and asked @faiq to not change.
Want to highlight this and make sure we're all ok with it.

(Sorry @faiq, in retrospect maybe its not that big of a deal changing these)

Reply:
I addressed that in the pr description when I opened it, happy to listen to any concerns but these changes will not affect users at this stage of the project, and I doubt they ever will. I think we could actually switch to kustomize now that I understand clusterctl deployment better (I think using helm to begin with was just because that was what I was familiar with...).